app/util/process_utils.py

```diff
@@ -48,3 +48,21 @@ def Popen_with_delayed_expansion(cmd, *args, **kwargs):
             cmd_with_deplayed_expansion.extend(cmd)
         cmd = cmd_with_deplayed_expansion
     return subprocess.Popen(cmd, *args, **kwargs)
+
+
+def get_environment_variable_setter_command(name, value):
+    """
+    Construct a platform specific command for setting an environment variable. Right now each command constructed
+    is designed to be chained with other commands.
+
+    :param name: The name of the environment variable
+    :type name: str
+    :param value: The value of the environment variable
+    :type value: str
+    :return: Platform specific command for setting the environment variable
+    :rtype: str
+    """
+    if os.name == 'nt':
+        return 'set {}={}&&'.format(name, value)
+    else:
+        return 'export {}="{}";'.format(name, value)
```

Comment:
Do you know if there is any benefit to embedding the env var setter into the command vs. using Popen's `env` param? I've wondered about that for awhile -- it would seem cleaner to use the `env` param.